src/test-records.ts

```diff
@@ -37,7 +37,7 @@ const validate = async (dir: string) => {
     const check_results: RdjsonLine[] = [];
 
     const linters = await _linters();
-    const files = glob.sync(`${dir}/*.json`);
+    const files = glob.sync(`${dir}/*`);
```

Comment:
I worried this might create a problem on systems that like to automatically place files in directories (`Thumbs.db`, `.DS_Store` etc.).

We should ignore everything in the `.gitignore` in this matcher. Unfortunately, our `glob` library doesn't support that as far as I can see. It might be worth switching to something like https://github.com/sindresorhus/globby.